.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -565,3 +568,117 @@ jobs:
         if: env.TEST_ON_X86_32 == 'true'
         run: ./test_wamr.sh ${{ env.X86_32_TARGET_TEST_OPTIONS }} ${{ matrix.test_option }} -t ${{ matrix.running_mode }}
         working-directory: ./tests/wamr-test-suites
+
+  test-wamr-ide:
```

Comment:
maybe add it to nightly_run instead of per-PR if it's 2hrs?

Reply:
LLDB version doesn't get updated that often, if we cache it (the first time a PR for the update is created), it should be fine

Reply:
I also think it should be fine. 

The LLDB [patch file](https://github.com/bytecodealliance/wasm-micro-runtime/commits/main/build-scripts/lldb-wasm.patch) was changes four times in the last 2 years.

Reply:
Thank you got it